### DISCUSSION TITLE
Error regarding bcrypt npm installation

### DISCUSSION THREAD

**Discussion by shivanjalikarande** [Note-Vault/Note-Vault#475] [Q&A] [answered]
while running command npm install showing error for bcrypt installation.

**Comment by VaibhavArora314** [comment]
Please also share the error @shivanjalikarande

**Comment by Ultimateutkarsh11** [comment, marked as answer]
Try updating npm to the latest version using the command: npm install -g npm
If the problem persists, you can use bcryptjs as an alternative to bycrypt: 
npm install bcryptjs
And in your JavaScript code, replace bcrypt with bcryptjs:
const bcrypt = require('bcryptjs');
